crates/aptos/src/common/types.rs

```diff
@@ -99,3 +152,100 @@ pub struct EncodingOptions {
     #[clap(long, default_value = "hex")]
     pub encoding: EncodingType,
 }
+
+#[derive(Debug, Parser)]
+pub struct PrivateKeyInputOptions {
```

Comment:
why not use an enum? or use clap options to make it so that these are exclusive?

Reply:
This is using clap options to make it exclusive?